src/components/SearchForm.js

```diff
@@ -1,36 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import "../styles/SearchForm.css";
-
-function SearchForm({ searchText, setSearchText, onSubmit }) {
-  // eslint-disable-next-line no-console
-  const handleInputChange = (event) => setSearchText(event.target.value);
-  /* At the moment, when text is entered into the text box, 
-  we have no way of referencing the value in the text box. 
-  To be able to use the value, we first need to capture it in the component's state. */
-  return (
-    <div className="search-form">
```

Comment:
This could do with a label tag:
`<label htmlFor="city-search>Search for a city</label>`